pkg/smart_wallet/config.go

```diff
@@ -0,0 +1,94 @@
+package smart_wallet
```

Comment:
Both [Google Go style guide](https://google.github.io/styleguide/go/decisions.html#package-names) and [Uber Golang style guide](https://github.com/uber-go/guide/blob/master/style.md#package-names) (which we used to follow) suggest that package names should not contain underscores `_`.

Considering that the repository is public, the possible solutions are as follows:
1. Change structure of packages. Since `session_key`, `userop`, and `smart_wallet` exist to complement each other in the same topic of ERC-4337 Account Abstraction, the refined structure can be `userop/{client,wallet,session,infra}`.
2. Change package name. The names I suggest are `smartwallet` or simply `sw`, though acronyms are not recommended. The names the style guidelines highlight to be wrong are `smartWallet` and `smart_wallet`. Same principles can be applied to `session_key` → `sessionkey` / `session`, `userop/testing/local_blockchain` → `userop/{testing|infra}/local`, and `userop/testing/public_blockchain` → `userop/{testing|infra}/public` packages.